main.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_ssm_document" "setup_lacework_agent" {
+  name          = "${var.aws_resources_prefix}setup-lacework-agent"
+  document_type = "Command"
+
+  target_type = "/AWS::EC2::Instance"
+
+  content = jsonencode({
+    schemaVersion = "2.2"
```

Comment:
I love this! I was worried about updates after the document has been created. Nicely done!